client/helpers/render-elements.helpers.tsx

```diff
@@ -26,49 +30,59 @@ const SIMPLE_TYPES = [
 // array of types such as banners, info, headings etc.
 const NOT_TO_BE_RENDERED = ['button', 'htmlelement', 'simplebuttonadvanced', 'simplecontent'];
 
+const MISC_LABELS_TO_REMOVE = ['Text/Images'];
+
 // some questions require wording changes on FE,
 // portalWording is a custom key containing the updated wording added by us, sent from CHEFS
 const getLabel = (component: any) => {
   return component.properties?.portalWording ?? component.label;
 };
 
-const MISC_LABELS_TO_REMOVE = ['primaryContactColumns', 'secondaryContactColumns', 'Text/Images'];
+const getValue = (componentKey: string, data: any, dataVal?: any) => {
+  if (dataVal !== undefined && dataVal !== NO_DATA_LABEL) return dataVal;
+
+  for (const [key, value] of Object.entries(data)) {
```

Comment:
what do you think about passing the container key at least? that way we don't need to check the entire data object each time?

Reply:
Yep, we're already passing it here: `componentKey: string`